src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/src/op_impl_check/single_op_graph.cpp

```diff
@@ -1242,6 +1242,13 @@ std::shared_ptr<ov::Model> generate(const std::shared_ptr<ov::op::v10::IsNaN> &n
     return std::make_shared<ov::Model>(results, ov::ParameterVector{param}, "is_nan_graph");
 }
 
+std::shared_ptr<ov::Model> generate(const std::shared_ptr<ov::op::v13::BitwiseNot> &node) {
+    const auto param = std::make_shared<ov::op::v0::Parameter>(ov::element::i32, ov::PartialShape{1, 2});
+    auto bitwise = std::make_shared<ov::op::v13::BitwiseNot>(param);
+    ov::ResultVector results{std::make_shared<ov::op::v0::Result>(bitwise)};
+    return std::make_shared<ov::Model>(results, ov::ParameterVector{param}, "bitwise_not_graph");
+}
```

Comment:
If the operation has UnaryElementwiseArithmetic as a base, it should be tested in conformance suite by extending this list:
https://github.com/openvinotoolkit/openvino/blob/7d1b7b6becf7b75d8d7f675d5262b1182235d90f/src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/src/op_impl_check/single_op_graph.cpp#L1340-L1343

Reply:
Reverted change to use UnaryElementswiseArithmetic as base due to differences in supported dtypes and possible problems and confusion. Now unary op would have op::Op as base, similarly to LogicalNot, and for binary bitwise ops, BinaryElementwiseBitwise will be added in separate PR.

Reply:
`BinaryElementwiseBitwise` cannot be used as it requires that operator transform two inputs into outputs.
`UnaryElementswiseArithmetic` has been proposed as shape inference use this base type calculate shape in common way.

It can be done later but maybe `UnaryElementswiseLogical` class has to be introduced and bot arithmetic and logica can share common class `UnaryElementswise` as shape inference can be done in same way.